app/views/users/show.html.erb

```diff
@@ -143,4 +156,4 @@
       </div>
```

Comment:
Du hast da mit diesem Commit zwar nichts dran geändert, aber kannst du start_date und end_date noch lokalisieren (also anstatt <%= schreibst du <%=l in Zeile 143 und 144